addons/godot-firebase/firebase/firebase.gd

```diff
@@ -17,27 +17,27 @@ const _AUTH_PROVIDERS : String = "firebase/auth_providers"
 
 ## @type FirebaseAuth
 ## The Firebase Authentication API.
-@onready var Auth := $Auth
+@onready var Auth : FirebaseAuth = $Auth
```

Comment:
Technically, the := gives us the types here and we shouldn't need them defined extra. It also shouldn't hurt anymore in 4.0, so I don't mind having it.

Reply:
Thought it helped with intellisense (to suggest the methods of our classes), but doesn't work which is pretty sad. Anyway it doesn't break stuff for sure.